config/initializers/ensure_env_vars_are_set.rb

```diff
@@ -1,7 +0,0 @@
-# Check up-front that neccessary ENV vars are present. This prevents delayed
-# errors, for example email failing on first execution because of missing configuration.
-# You can add these variables in development or CI using your .env file.
-
-%w(SECRET_KEY_BASE).each do |key|
-  ENV.fetch(key) { raise "#{key} not found in ENV" }
-end
```

Comment:
There was another initializer `01_check_for_presence_of_required_env_vars.rb` which was doing the same thing, so this one removed